flambe/nlp/language_modeling/datasets.py

```diff
@@ -96,14 +91,75 @@ def _process(self, file: bytes) -> List[Tuple[str]]:
         """
         decoded_text = file.decode('utf-8')
         # Replace end of line tokens
-        if self.eol is not None:
+        if self.eol is not None and not self.split_by_sentence:
             decoded_text = decoded_text.replace('\n', self.eol)
 
         # Split by sentence or unroll
         if self.split_by_sentence:
             text = [(sent.strip(),) for sent in nltk.tokenize.sent_tokenize(decoded_text)]
         else:
-            steps = range(0, len(decoded_text) - self.unroll_size, self.unroll_size)
-            text = [(" ".join(decoded_text[i:i + self.unroll_size]),) for i in steps]
+            text = [(decoded_text,)]
 
         return text
+
+
+class Enwiki8(TabularDataset):
+    """The official WikiText103 dataset."""
+
+    ENWIKI_URL = "http://mattmahoney.net/dc/enwik8.zip"
+
+    def __init__(self,
+                 unroll_size: Optional[int] = 100,
+                 num_test_symbols: int = 5000000,
```

Comment:
What does `5000000` stand for? Forgot to add documentation for a couple of parameters (including `num_test_symbols` )

Reply:
Fixed